scripts/opentitan/run-bazel-tests.sh

```diff
@@ -94,24 +97,20 @@ unexpected_passes="$(comm -23 "$passed" "$expected")"
 status=0
 
 if [ -n "$unexpected_failures" ]; then
-  echo >&2
-  echo >&2 Tests that we expected to pass which did NOT pass:
-  echo "$unexpected_failures" | awk '$0="  "$0' >&2
+  echo                                                      | tee -a "$GITHUB_STEP_SUMMARY" >&2
+  echo "Tests that we expected to pass which did NOT pass:" | tee -a "$GITHUB_STEP_SUMMARY" >&2
+  echo "$unexpected_failures" | awk '$0="- "$0'             | tee -a "$GITHUB_STEP_SUMMARY" >&2
```

Comment:
As a very minor nit (personal preference, honestly) I would like it if these test names were inline code via backticks so that they were monospaced.

```suggestion
  echo "$unexpected_failures" | awk '$0="- `"$0"`"'             | tee -a "$GITHUB_STEP_SUMMARY" >&2
```

Feel free to ignore :)